tests/h/schemas/annotation_test.py

```diff
@@ -625,18 +625,19 @@ def test_it_returns_only_known_params(self, schema):
 
         assert params == expected_params
 
-    def test_it_handles_duplicate_keys(self, schema):
-        expected_params = MultiDict(
-            [("url", "http://foobar"), ("url", "http://foobat")]
-        )
+    @pytest.mark.parametrize(
+        "key,examples",
+        (
+            ("url", ["http://foobar", "http://foobat"]),
+            ("group", ["inu3340958u", "97jdm5o457"]),
+        ),
+    )
+    def test_it_handles_duplicate_keys(self, schema, key, examples):
+        expected_params = MultiDict([(key, example) for example in examples])
         input_params = deepcopy(expected_params)
-        input_params.add("unknownparam", "foo")
-        input_params.add("unknownparam", "bar")
```

Comment:
We test this separately in a massive test called "we discard unknowns" or something similar. Not sure why this was here.